.woodpecker.yml

```diff
@@ -22,10 +25,24 @@ steps:
 
   leptos_fmt:
     image: *rust_image
+    group: format
     commands:
-      - cargo install leptosfmt
+      - wget https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-unknown-linux-musl.tgz
```

Comment:
Why musl? Also, is there a way to avoid needing to download and extract binstall each time it is used?